lib/reactotron-mst/src/reactotron-mst.ts

```diff
@@ -181,7 +182,7 @@ export function mst(opts: MstPluginOptions = {}) {
         const modelType = getType(node)
 
         // we only want types
-        if (modelType.isType) {
+        if (isType(modelType)) {
```

Comment:
This seems correct and does work, but want to make sure it is what we actually want.

Reply:
Looks fine to me.